### ISSUE TITLE
[a11y Attend Meeting] Design Intent: Appointments FE Team - Appointment Details re-design

### ISSUE THREAD

**Issue by ohgreatkate** [department-of-veterans-affairs/va.gov-team#65756]
**Meeting Info:**  Monday, September 18, 3:30 – 4:00pm

- [GH ticket](https://www.google.com/url?q=https://app.zenhub.com/workspaces/appointments---productdesign-5fff340c2d80a4000fb6f69c/issues/gh/department-of-veterans-affairs/va.gov-team/64825&sa=D&source=calendar&ust=1695486051698291&usg=AOvVaw1KGrOB3GAR53ZbMPrRFAo6)
- [Slack request](https://www.google.com/url?q=https://dsva.slack.com/archives/CMNQT72LX/p1694449766414639&sa=D&source=calendar&ust=1695486051698291&usg=AOvVaw3vaRcXKY93SQLgL1Ydk1Li)
- [Join Zoom Meeting](https://oddball-io.zoom.us/j/85753669580?pwd=ZGVLdklsSGVnVWN0ejNlYjVvRXVxUT09)

Meeting notes in Google Drive: [Appointments FE Folder](https://drive.google.com/drive/folders/1SVlILgBvhtkKtEVJIEyDoskqM0K4AQM8?usp=drive_link)

**Comment by ohgreatkate** [comment]
**### Design intent Appointments FE key takeaways:**

- if buttons are near each other think about their actions and if they are truly friends or if an adjustment in layout would help with cognition on the two actions.

- for screen readers: think about where the buttons and their actions are in relation to the last heading, is this where you would go looking for that action?

- think about alerts vs status and how to style them.

- add to calendar feature available on past and canceled appointments? what about 3o minutes before? at what point is this CTA not useful
